certified-connectors/Cascade Strategy/apiDefinition.swagger.json

```diff
@@ -310,20 +257,77 @@
             }
           }
         },
-        "summary": "Update Measure Historical Value",
-        "description": "Updates historical values of the selected measure.",
-        "x-ms-visibility": "important",
-        "operationId": "UpdateMeasureHistoricalValue",
-        "consumes": ["application/json"],
+        "summary": "Update Measure",
+        "operationId": "UpdateMeasure",
+        "consumes": [
+          "application/json"
+        ],
         "parameters": [
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "workspace_id": {
+                  "type": "string",
+                  "description": "Select Workspace of the measure to be updated.",
+                  "title": "Workspace",
+                  "x-ms-dynamic-values": {
+                    "operationId": "GetWorkspaces",
```

Comment:
The path template has been changed for the operation. ValuePath: operation/GetWorkspaces, can you please confirm that no existing flows or users will be affected by this change?